packages/wonder-blocks-button/components/button-core.js

```diff
@@ -52,14 +53,15 @@ export default class ButtonCore extends React.Component<Props> {
             size === "small" && sharedStyles.small,
         ];
 
-        const Tag = href ? StyledAnchor : StyledButton;
+        const Tag = tag || (href ? StyledAnchor : StyledButton);
         const Label = size === "small" ? LabelSmall : LabelLarge;
         return (
             <Tag
                 style={[defaultStyle, style]}
                 disabled={disabled}
                 data-test-id={testId}
                 href={href}
+                to={href}
```

Comment:
This is going to cause problems if we're just using a normal `a` or `button`, React will complain about properties that it doesn't know how to handle.